ethpm_cli/commands/registry.py

```diff
@@ -203,3 +216,15 @@ def generate_registry_store_data(
         "alias": alias,
         "active": activate,
     }
+
+
+def display_packages(all_package_names: Iterable[str], config: Config) -> None:
+    for package_name in all_package_names:
+        all_releases = config.w3.pm.get_all_package_releases(package_name)
+        cli_logger.info(f"Retrieving all releases for {bold_blue(package_name)}: \n")
```

Comment:
I'd put line 224 above 223 - since the call to w3 to fetch all of the releases could take a couple of seconds, it would be better to have this message displayed before that call is made.